Microsoft.AspNet.NodeServices/HostingModels/OutOfProcessNodeInstance.cs

```diff
@@ -103,18 +111,18 @@ public OutOfProcessNodeInstance(string entryPointScript, string commandLineArgum
                     }
                 }
             };
-                                
+
             this._nodeProcess.BeginOutputReadLine();
-            this._nodeProcess.BeginErrorReadLine();            
+            this._nodeProcess.BeginErrorReadLine();
         }
-        
+
         protected virtual void OnBeforeLaunchProcess() {
         }
 
         protected virtual void OnOutputDataReceived(string outputData) {
             Console.WriteLine("[Node] " + outputData);
         }
-        
```

Comment:
It was just a random comment, sorry. I only saw a demo of this thing during the MVP summit and it was very cool so I got interested. @SteveSanderson needs to look at it I think.